### DISCUSSION TITLE
How to make the belt attribution more festive?

### DISCUSSION THREAD

**Discussion by nelsonmestevao** [coderdojobraga/gongo#7] [Q&A]
At the time of the final ceremony, we usually announce the raising of ninja belts. At online sessions, we are all in the same discord voice channel (unless we are so many that the room is too small).

In those cases, it would be cool if the bot played a [gongo sound](https://www.youtube.com/watch?v=0PJcItAhIOA) in the general voice channel when some mentor uses the command bot :tada: 

What are other ways to make this moment more festive, even if we are not in a voice channel?

**Comment by ruioliveira02** [comment]
Message the ninja to congratulate them, and remind them of what the next belt is and the requirements are, for example
